config/main.py

```diff
@@ -1729,7 +1730,7 @@ def portchannel(ctx, namespace):
 
     config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=str(namespace))
     config_db.connect()
-    ctx.obj = {'db': config_db, 'namespace': str(namespace)}
+    ctx.obj = {'db': config_db, 'namespace': str(namespace), 'db_wrap': ctx.obj}
```

Comment:
@vivekreddynv why do we need this?

Reply:
ctx.obj is originally of set to utilities_common.Db() type in the config() method. That is overwritten by this custom dict. 

`ctx.obj = {'db': config_db, 'namespace': str(namespace)}` but for the connection pool to work, i need the SonicV2Connector() object declared in Db() class thus this addition

Reply:
@vivekreddynv imho, better to reuse: https://github.com/Azure/sonic-utilities/blob/master/utilities_common/cli.py#L19

Reply:
Thanks for the suggestion, i'll use this

Reply:
handled